### DISCUSSION TITLE
how to set default value for Autocomplete (material ui) when set default value is async

### DISCUSSION THREAD

**Discussion by publicaccount365** [react-hook-form/react-hook-form#2878] [Q&A]
how to set default value for Autocomplete (material ui) when set default value is async

https://codesandbox.io/s/material-demo-forked-9x579?file=/demo.js:925-937

**Comment by bluebill1049** [comment]
take a look at the reset() api and useform defaultValues section

**Reply by bluebill1049** [reply]
https://react-hook-form.com/api#useForm
